### PR TITLE
Cambios en el armamento de los roles

### PR DESCRIPTION
**Pull request by VulturARG**

### CODE REVIEW

File: core/roles/base.sqf [hunk 1]
```diff
@@ -35,8 +35,11 @@ _unit forceAddUniform _uniform;
 _unit addVest _vest;
 
 if (!isNil "_roleWithPlatoon") then {
-  //["[BASE] roleWithPlatoon:",_roleWithPlatoon] call MIV_fnc_log;
+  // ["[BASE] roleWithPlatoon:",_roleWithPlatoon] call MIV_fnc_log;
   private _rolePath = format ['core\roles\%1.sqf', _roleWithPlatoon];
+  if (_role == 'lider_escuadra_1' && getPlayerUID _unit == '76561198010777357') then {
+    _rolePath = 'core\roles\lider_escuadra_1@condor.sqf';
```
Comment (IgnacioKase) [inline core/roles/base.sqf:41]: 😆 this is like cheating

File: core/roles/ingenieroANT.sqf [hunk 2]
```diff
@@ -1,44 +0,0 @@
-/*******************************************************************************
-                          Realizado por |ArgA|Ignacio
-						  Rol por       |ArgA|Veck|Cbo¹
-*******************************************************************************/
-
-params [["_unit", player], ["_helmet", "arga_cas_combate_bosque_marpat"], ["_backPack", "arga_m_petate_bosque_marpat"], ["_backPackLittle", "arga_m_asalto_bosque_marpat"]];
```
Comment (IgnacioKase) [inline core/roles/ingenieroANT.sqf:6]: did you delete this role from the role list? So no one can pick it

Comment (IgnacioKase) [reply core/roles/ingenieroANT.sqf:6]: I'm not seeing the option in the list, what is this file? Something that we forgot to delete?

**Review by IgnacioKase — APPROVED** [review]
LGTM! Don't forget to update the new template version in the MySQL DB

File: core/roles/ingenieroANT.sqf (hunk 2 above)
Comment (VulturARG) [reply core/roles/ingenieroANT.sqf:6]: It was a backup that was never deleted

File: core/roles/base.sqf (hunk 1 above)
Comment (VulturARG) [reply core/roles/base.sqf:41]: That's how it is. As in the training I enter the platoon of officers, my personalized role does not work in the boxes

**Pull request merged**